.github/dependabot.yml

```diff
@@ -1,69 +1,14 @@
 ---
 version: 2
 updates:
-  #TODO: Replace go directories with wildcard https://github.com/dependabot/dependabot-core/issues/2178
   - package-ecosystem: "gomod"
-    directory: "/processor/lsmintervalprocessor/"
-    schedule:
-      interval: "daily"
-    labels:
-      - automation
-    groups:
-      otel-dependencies:
-        patterns:
-          - "go.opentelemetry.io/*"
-          - "github.com/open-telemetry/opentelemetry-collector-contrib/*"
-    ignore:
-      - dependency-name: "*"
-        update-types: ["version-update:semver-major"]
-
-  - package-ecosystem: "gomod"
-    directory: "/processor/elasticinframetricsprocessor/"
-    schedule:
-      interval: "daily"
-    labels:
-      - automation
-    groups:
-      otel-dependencies:
-        patterns:
-          - "go.opentelemetry.io/*"
-          - "github.com/open-telemetry/opentelemetry-collector-contrib/*"
-    ignore:
-      - dependency-name: "*"
-        update-types: ["version-update:semver-major"]
-
-  - package-ecosystem: "gomod"
-    directory: "/processor/elastictraceprocessor/"
-    schedule:
-      interval: "daily"
-    labels:
-      - automation
-    groups:
-      otel-dependencies:
-        patterns:
-          - "go.opentelemetry.io/*"
-          - "github.com/open-telemetry/opentelemetry-collector-contrib/*"
-    ignore:
-      - dependency-name: "*"
-        update-types: ["version-update:semver-major"]
-
-  - package-ecosystem: "gomod"
-    directory: "/processor/ratelimitprocessor/"
-    schedule:
-      interval: "daily"
-    labels:
-      - automation
-    groups:
-      otel-dependencies:
-        patterns:
-          - "go.opentelemetry.io/*"
-          - "github.com/open-telemetry/opentelemetry-collector-contrib/*"
-    ignore:
-      - dependency-name: "*"
-        update-types: [ "version-update:semver-major" ]
-
-  - package-ecosystem: "gomod"
-    directory: "/connector/signaltometricsconnector"
+    directories:
+      - "/processor/lsmintervalprocessor/"
+      - "/processor/elasticinframetricsprocessor/"
+      - "/processor/elastictraceprocessor/"
+      - "/processor/ratelimitprocessor/"
+      - "/connector/signaltometricsconnector"
+      - "/internal/tools/"
```

Comment:
Can we use wildcards here?

Reply:
Also, I don't quite get what this PR will reduce, IIUC, we would still get as many PRs as the number of components or am I mistaken? (I would not want a single PR with all components as otel it is quite common for components to require fixes due to upgrade of otel versions and if multiple components break in a single PR then it would be quite a challenge to fix that in addition to the challenge with different teams having accountability for different components).